lib/reek/examiner.rb

```diff
@@ -10,7 +10,7 @@ module Reek
   #
   # @public
   #
-  # :reek:TooManyInstanceVariables: { max_instance_variables: 6 }
+  # :reek:TooManyInstanceVariables: { max_instance_variables: 7 }
```

Comment:
That's because of the memoization in "#run" below. We really should fix the detector to not count this as instance variables.